src/app/components/Curation/CurationPromo/index.stories.tsx

```diff
@@ -1,57 +1,45 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { withServicesKnob } from '#psammead/psammead-storybook-helpers/src';
-import { withKnobs } from '@storybook/addon-knobs';
 import fixture from '#data/pidgin/topics/c95y35941vrt.json';
 import { MEDIA_TYPES } from '#components/Promo';
-import { ServiceContextProvider } from '../../../contexts/ServiceContext';
-import ThemeProvider from '../../ThemeProvider';
+import withServicesDecorator from '#storybook/withServicesDecorator';
 import Promo from '.';
-import { StoryProps } from '../../../models/types/storybook';
 
 const Wrapper = styled.div`
   max-width: 30rem;
 `;
 
-const Component = ({ service, variant }: StoryProps) => {
+const Component = () => {
   return (
-    <ThemeProvider service={service} variant={variant}>
-      <ServiceContextProvider service={service} variant={variant}>
-        <Wrapper>
-          <Promo {...fixture.data.curations[0].summaries[0]} />
-        </Wrapper>
-      </ServiceContextProvider>
-    </ThemeProvider>
+    <Wrapper>
+      <Promo {...fixture.data.curations[0].summaries[0]} />
+    </Wrapper>
   );
 };
 
 // eslint-disable-next-line react/prop-types
-const WithMediaIndicator = ({ service, variant }: StoryProps) => {
+const WithMediaIndicator = () => {
   return (
-    <ThemeProvider service={service} variant={variant}>
-      <ServiceContextProvider service={service} variant={variant}>
-        <Wrapper>
-          <Promo
-            {...fixture.data.curations[0].summaries[0]}
-            type={MEDIA_TYPES.VIDEO}
-            duration={123}
-          />
-          <Promo
-            {...fixture.data.curations[0].summaries[0]}
-            type={MEDIA_TYPES.PHOTO_GALLERY}
-            duration={123}
-          />
-        </Wrapper>
-      </ServiceContextProvider>
-    </ThemeProvider>
+    <Wrapper>
+      <Promo
+        {...fixture.data.curations[0].summaries[0]}
+        type={MEDIA_TYPES.VIDEO}
+        duration={123}
+      />
+      <Promo
+        {...fixture.data.curations[0].summaries[0]}
+        type={MEDIA_TYPES.PHOTO_GALLERY}
+        duration={123}
+      />
+    </Wrapper>
   );
 };
 
 export default {
-  title: 'New Components/Curation/Promo - Normal',
+  title: 'Components/Curation/Promo - Normal',
   Component,
-  decorators: [withKnobs, withServicesKnob()],
   parameters: { chromatic: { disable: true } },
+  decorators: [withServicesDecorator],
```

Comment:
Should this be removed, as it is available for all stories by default?

Reply:
This can be removed from this story as the global service control isn't needed for this one.